reporter-web-app/scripts/start.js

```diff
@@ -54,18 +55,27 @@ if (process.env.HOST) {
   console.log(
     `If this was unintentional, check that you haven't mistakenly set it in your shell.`
   );
-  console.log(`Learn more here: ${chalk.yellow('http://bit.ly/2mwWSwH')}`);
+  console.log(
```

Comment:
Shouldn't these be combined to a single `log` call with `\n` n between?

Reply:
start.js comes from upstream create-react-app